### DISCUSSION TITLE
Task09_Prob2

### DISCUSSION THREAD

**Discussion by rockyx50** [chulminy/CIVE497-CIVE700#140] [Q&A]
I created a new dataset for this problem with the only main difference being that the data values range from 0:100 instead of -6:6. Whenever I run the NN to classify the two data sets it seems as if the classification can only be done along a diagonal line that must go through 0,0. The boundary line between the two datasets never changes orientation, its always diagonal in the same direction. Is there something else besides the data that I need to change?

![image](https://user-images.githubusercontent.com/60040283/115771651-36776a80-a37c-11eb-928a-3990222a1321.png)

**Comment by chulminy** [comment]
It looks the data points for drawing the boundary are too sparse. How about setting up 0:0.01:100?

**Reply by rockyx50** [reply]
I change the meshgrid to 0:1:100 and deviation between the data points to be less and got the result below. However the boundary line is still only diagonal. Was that what you meant by making it less sparse? 
![image](https://user-images.githubusercontent.com/60040283/115781820-e81c9880-a388-11eb-83fa-045384b9db61.png)

**Reply by chulminy** [reply]
Your dataset is too easy. Can you try out more complex one? I haven't checked your code yet but intuitively, that diagonal line is the simplest model that can divide them with 100% accuracy. There is no reason for NN to develop more complex model.

**Reply by rockyx50** [reply]
Okay i changed my dataset and the demo code I was using and got it to work. Thanks!
